frontend/src/metabase/visualizations/shared/components/RowChart/utils/data.ts

```diff
@@ -26,10 +26,6 @@ export const calculateNonStackedBars = <TDatum>(
         const xValue = series.xAccessor(datum);
         const isNegative = xValue != null && xValue < 0;
 
-        if (xValue == null) {
-          return null;
-        }
```

Comment:
Then `.filter(isNotNull)` after this list might not be needed since it won't return null?